wasmtest/run-wasm-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+set -x
+
+# install build tools
+if ! which rustup ; then
+	curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly 
+	source $HOME/.cargo/env
+fi
+rustup target add wasm32-unknown-unknown
+which ontio-wasm-build || cargo install --git=https://github.com/ontio/ontio-wasm-build
```

Comment:
this code setup move it to contracts-rust better?

Reply:
`ontio-wasm-build` is used below, I prefer group the setup and usage   together.